src/sentry/utils/integrationdocs.py

```diff
@@ -111,6 +114,45 @@ def sync_docs():
     for platform_id, platform_data in iteritems(data['platforms']):
         for integration_id, integration in iteritems(platform_data):
             sync_integration_docs(platform_id, integration_id, integration['details'])
+            python_platforms.append(get_integration_id(platform_id, integration_id))
+
+    # create python platform constants
+    __sync_platform_types(python_platforms)
+
+
+def __sync_platform_types(platforms):
+    """
+    Updating existing platform types whenever docs are synced
+    """
+    # Cannot use six here
+    valid_platforms = set(platform for platform in platforms)
+
+    echo("updatng platform constants")
+    with open('platform-constants.txt', 'w') as file:
+        pickle.dump(valid_platforms, file)
+
+
+def get_platform_types():
+    try:
+        with open('platform-constants.txt', 'r') as file:
+            return pickle.load(file)
+    except BaseException:
+        # Fall back onto a hardcoded version of the file.
```

Comment:
I have no idea what the best way to go about the system not being run before this function is called. 

@denamwangi when are these functions typically called